### PR TITLE
ShipDesignAI code cleanup

### PR DESCRIPTION
**Pull request by Morlic-fo**

### CODE REVIEW

File: default/AI/ShipDesignAI.py [hunk 1]
```diff
@@ -812,8 +814,10 @@ def update_stats(self, ignore_species=False):
         self.detection = 0  # TODO: Add self.hull.detection once available in interface
         self.shields = 0    # TODO: Add self.hull.shields if added to interface
         self.troops = 0     # TODO: Add self.hull.troops if added to interface
-        self.production_cost = local_cost_cache.get(self.hull.name, self.hull.productionCost(fo.getEmpire().empireID, self.pid))
-        self.production_time = local_time_cache.get(self.hull.name, self.hull.productionTime(fo.getEmpire().empireID, self.pid))
+        self.production_cost = local_cost_cache.get(self.hull.name,
+                                                    self.hull.productionCost(fo.getEmpire().empireID, self.pid))
+        self.production_time = local_time_cache.get(self.hull.name,
+                                                    self.hull.productionTime(fo.getEmpire().empireID, self.pid))
```
Comment (Cjkjvfnby) [inline default/AI/ShipDesignAI.py]: You can use `fo.empireID()`

**Comment by Vezzra** [comment]
I was going to take a look at this, test it and eventually merge it (as @Dilvish-fo isn't around much ATM). Why did you close it? Something wrong with it, or do you intend to provide an enhanced version?

**Comment by Morlic-fo** [comment]
Oops, seems when cleaning up my branches I accidentally deleted this one as well. Was not supposed to be closed.

**Comment by Vezzra** [comment]
Ah, ok. Well, a bit late now for me today, will try to get to it next weekend (if no one beats me to it :wink:)

**Comment by geoffthemedio** [comment]
bump

**Comment by Vezzra** [comment]
Oh, oops, forgot - sorry.

Ok, builds and runs fine on OSX. As far as I can tell the code changes look innocent enough, and assuming you, @Morlic-fo, did some basic testing yourself I'm going to merge this.

**Pull request merged**